cmd/gctcli/commands.go

```diff
@@ -391,10 +383,6 @@ func getExchangeOTPCode(c *cli.Context) error {
 		exchangeName = c.Args().First()
 	}
 
```

Comment:
Same with here. You'll receive a generic error `"exchange does not have a OTP secret stored"` which doesn't say its not loaded